lib/bolt/transport/powershell.rb

```diff
@@ -195,7 +195,7 @@ def shell_init
   $propbag = @{}
   foreach ($name in Get-Member -InputObject $xml -MemberType Properties | Where-Object{$_.Name -notmatch "^__|\\btype\\b"} | Select-Object -ExpandProperty name) {
     Write-Debug "$Name Type: $($xml.$Name.type)" -Debug:$false
-    $propbag."$Name" = Convert-Properties $xml."$name"
+    if ("$($xml.$Name)" -eq "System.Xml.XmlElement") { $propbag."$Name" = Convert-Properties $xml."$name" }
```

Comment:
I want to do something like `if ( $xml.$Name -is System.Xml.XmlElement )` but that seems to be wrong. Converting to the string representation is just a hack to show the idea.

Reply:
You want to make sure the object instance is a `System.Xml.XmlElement`? Is that necessary?

You'd do that like:

```powershell
if ($xml -is System.Xml.XmlElement)
```

Reply:
I forgot to put the brackets on earlier that you need to qualify a .NET type in PowerShell. Comment ^ updated

Reply:
Thanks, `[Xml.XmlElement]` did the trick, but of course we ditched that idea :rofl: